src/ln/channelmonitor.rs

```diff
@@ -1860,31 +1881,28 @@ impl ChannelMonitor {
 				if input.previous_output.txid == current_local_signed_commitment_tx.txid {
 					scan_commitment!(current_local_signed_commitment_tx.htlc_outputs.iter().map(|&(ref a, _, _)| a),
 						current_local_signed_commitment_tx.htlc_sources,
-						"our latest local commitment tx");
+						"our latest local commitment tx", true);
 				}
 			}
 			if let Some(ref prev_local_signed_commitment_tx) = self.prev_local_signed_commitment_tx {
 				if input.previous_output.txid == prev_local_signed_commitment_tx.txid {
 					scan_commitment!(prev_local_signed_commitment_tx.htlc_outputs.iter().map(|&(ref a, _, _)| a),
 						prev_local_signed_commitment_tx.htlc_sources,
-						"our previous local commitment tx");
+						"our previous local commitment tx", true);
 				}
 			}
 			if let Some(&(ref htlc_outputs, ref htlc_sources)) = self.remote_claimable_outpoints.get(&input.previous_output.txid) {
-				scan_commitment!(htlc_outputs, htlc_sources, "remote commitment tx");
+				scan_commitment!(htlc_outputs, htlc_sources, "remote commitment tx", false);
 			}
 
 			// If tx isn't solving htlc output from local/remote commitment tx and htlc isn't outbound we don't need
 			// to broadcast solving backward
```

Comment:
nit: Maybe change the term "broadcast" to "relay" or other, I mean we should enforce terminology to dissociate "broadcasting tx onchain" from to "relay state offchain" ?

Reply:
Done.